packages/terra-button/src/_themes.scss

```diff
@@ -0,0 +1,84 @@
+@import './mixins';
+
+@include cerner-consumer-theme {
+  --terra-consumer-color: #64696c;
+  --terra-consumer-color-dark-30: #383f42;
+  --terra-consumer-color-dark-50: #1c1f21;
+  --terra-consumer-color-light-30: rgba(#383f42, 0.3);
+  --terra-consumer-color-light-50: rgba(#64696c, 0.5);
+  --terra-consumer-color-light-65: rgba(#383f42, 0.65);
```

Comment:
not sure how i feel about the numbers... I'd almost prefer primary, primary-dark, secondary, secondary-dark or something.